utils/cgroupfs-mount/files/cgroupfs-mount.init

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh /etc/rc.common
+
+USE_PROCD=1
+START=1
+
+start_service() {
+	procd_open_instance
+
+	# Procd mounts non-hierarchical cgroupfs so unmount first before cgroupfs-mount
+	procd_set_param command { mountpoint -q /sys/fs/cgroup && umount /sys/fs/cgroup/; cgroupfs-mount }
```

Comment:
procd is for long-running services.  So there is no need to `USE_PROCD` in this case.

Reply:
Changed, also using `boot()` now as it only needs to happen on startup